ci/container_scripts/test.sh

```diff
@@ -39,6 +41,9 @@ FLAGS+=("-E" "$EXCLUDE")
 # Pass through an optional config-name, which can enable more tests
 FLAGS+=("-C" "$CONFIG")
 
+# Exclude tor tests as we test them in a different workflow
+FLAGS+=("-LE" "tor")
```

Comment:
Optional nit: prefer writing the long version of the flags if they exist (I don't remember what L and E mean).

Reply:
Done.